tao_compiler/mlir/disc/tests/regression/reshape_fusion_cpu.cc

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tensorflow/compiler/mlir/disc/tests/mlir_feature_test.h"
+#include "tensorflow/compiler/mlir/disc/tests/mlir_test.h"
+#include "tensorflow/core/platform/test.h"
+#include "tensorflow/core/util/env_var.h"
+
+namespace mlir_test {
+
+const std::string c_ft_path =
+    "tensorflow/compiler/mlir/disc/tests/regression/data/";
+
+TEST(ReshapeFusionCPUTest, kLoopTest) {
+  setenv("DISC_ENABLE_STITCH", "true", 1);
```

Comment:
Just curious, have we support `stitch` fusion on cpu devices?

Reply:
Stitch on cpu is implemented on CPU in the end of 2021.  Here is an introducing material (start from page 16).
https://bladedisc.oss-cn-hangzhou.aliyuncs.com/docs/performance-optimization-practice.pdf

reshape like op support in stitch will be implemented in next PR.